### DISCUSSION TITLE
Is there a way to use the release versions of packages instead of the git versions?

### DISCUSSION THREAD

**Discussion by Quadragintillion** [end-4/dots-hyprland#695] [Q&A]
I've been recently having an issue with all Hyprland programs that causes them to fail to start:
`Hyprland: error while loading shared libraries: libhyprutils.so.0: cannot open shared object file: No such file or directory`
Git versions of packages are going to be extremely unstable, and I do not want that. I would much prefer to just have the latest actual release, as it is less prone to issues like this. Can I downgrade to use the releases instead of the git versions?

**Comment by Quadragintillion** [comment]
Normally I could just run `sudo pacman -S <main-package>` but `illogical-impulse-widgets` explicitly requires the git versions.

**Reply by end-4** [reply]
in arch-packages/illogical-impulse-widgets/PKGBUILD edit the 'depends='

**Reply by Quadragintillion** [reply]
> in arch-packages/illogical-impulse-widgets/PKGBUILD edit the 'depends='

Do I have to entirely reinstall the dots to apply this?

**Comment by hrsvrn** [comment]
I kind of agree on this tbh.. git releases sometimes don't work like hyprutils. Evrything else works like a glove
